test/utils/images/images.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import (
+	"os"
+
+	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
+)
+
+var testImage string
```

Comment:
besides my general dislike for globals and `init` (unless where is clearly needed, and here is not), we can simply have something like
```
func GetPauseImage() string {
       if pullSpec, ok := os.LookupEnv("E2E_PAUSE_IMAGE_URL"); ok {
               return pullSpec
       }
       return objects.PauseImage
}
```
and remove everything else